### PR TITLE
Update February 2020

### PR DESCRIPTION
**Pull request by Liliththerebel**
I hope everything is right, 
in case of any doubt do not hesitate to contact me.
Regards

### CODE REVIEW

File: src/assets/i18n/it.po [hunk 1]
```diff
@@ -798,7 +814,8 @@ msgstr ""
 msgid ""
 "Add a passphrase to the new  encryption key. This is used to lock and "
 "unlock the pool."
-msgstr ""
+msgstr "Aggiungi una passphrase alla nuova chiave crittografica. Questa è usata per bloccare e "
+sbloccare il pool."
```
Comment (erinix) [inline src/assets/i18n/it.po]: you appear to have a missing begin quote here, please fix

File: src/assets/i18n/it.po [hunk 2]
```diff
@@ -1785,49 +1822,58 @@ msgid ""
...
+"files a cui accedere tranite WebDAV deve essere impostata  manualmente su"
+"<samp>webdav</samp> o <samp>www</samp> user/group."
 
 msgid ""
 "Change from <i>public</i> to increase system security. Can only contain "
 "alphanumeric characters, underscores, dashes, periods, and spaces. This can "
 "be left empty for <i>SNMPv3</i> networks."
 msgstr ""
+"Passa da <i>public</i> per aumentare il livello di sicurezza. Può contenere solo "
+caratteri alfanumerici, trattini bassi, trattini, punti, e spazi. Questo può "
```
Comment (erinix) [inline src/assets/i18n/it.po]: also here

**Comment by erinix** [comment]
there also appear to be merge conflicts

**Comment by erinix** [comment]
looks like you fixed the breakages but there is still a merge conflict, please fix it so I can merge

**Comment by Liliththerebel** [comment]
Should I make a new pull request after fixing breakage? 
Regards

**Comment by erinix** [comment]
The merge conflict doesn't look very complicated, you could probably just fix with the web editor

**Comment by erinix** [comment]
if it is easier for you to make a new PR with these same things feel free to do so, I would try to fix it for you but it says it will push to your master branch and I don't want to mess with that.  I would suggest making new branches whenever you make a PR

**Comment by erinix** [comment]
I created a separate PR that includes these changes in it and the current PO file, ssee https://github.com/freenas/webui/pull/3631

**Pull request closed without merging**